diagnostic.md

```diff
@@ -150,7 +182,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+a line of divs shown from left to right as divTwo, divThree, divFive, divOne,
+and divFourwill be on the left side of the next line.
```

Comment:
Close! 
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5